utils/error.go

```diff
@@ -1,13 +1,13 @@
 package utils
 
 import (
-	"log"
 	"runtime/debug"
+
+	"go.uber.org/zap"
 )
 
 func MustNil(e error) {
 	if e != nil {
-		debug.PrintStack()
-		log.Fatalf("error occured %v", e)
+		zap.L().Fatal("error occurred", zap.Error(e), zap.String("trace", string(debug.Stack())))
```

Comment:
Since `MustNil` is only used in tests, which print stack info and shouldn't be interacted with user, we can move this func into test package.

Reply:
done